pkg/virtctl/templates/templates.go

```diff
@@ -52,7 +52,9 @@ func ExactArgs(nameOfCommand string, n int) cobra.PositionalArgs {
 	return func(cmd *cobra.Command, args []string) error {
 		if len(args) != n {
 			fmt.Printf("fatal: Number of input parameters is incorrect, %s accepts %d arg(s), received %d\n\n", nameOfCommand, n, len(args))
-			cmd.Help()
+			if err := cmd.Help(); err != nil {
+				panic("Help doesn't return error")
```

Comment:
1) The error message is not clear here.  
2) Why panic and not returning error ?

Reply:
I probably didn't find the return statement in the Help() function, I will append this error to one which is below.